examples/BinaryEntityCreation/main.c

```diff
@@ -95,16 +95,20 @@ int main(
                     UXR_REPLACE);
 
     uxrObjectId datawriter_id = uxr_object_id(0x01, UXR_DATAWRITER_ID);
+    uxrQoS_t qos = {
+        .reliability = UXR_RELIABILITY_RELIABLE, .durability = UXR_DURABILITY_TRANSIENT_LOCAL,
+        .history = UXR_HISTORY_KEEP_LAST, .depth = 1
```

Comment:
```suggestion
        .history = UXR_HISTORY_KEEP_LAST, .depth = 0
```

In this example, `depth = 0`